community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/main.tf

```
@@ -205,42 +188,52 @@ resource "google_storage_bucket_object" "devel" {
 # SCRIPTS #
 ###########
 
-resource "google_storage_bucket_object" "controller_startup_scripts" {
-  for_each = {
+locals {
+  ok_chars = "/[^a-zA-Z0-9-_]/"
```

```suggestion
  bad_chars = "/[^a-zA-Z0-9-_]/"
```
Actually this regexp matches all the characters you don't want in the filename.